### PR TITLE
Doruk/get followers

### PR DESCRIPTION
**Pull request by ddemirci**
Routing has been changed but can be refactored as you wish.

**Get Followers:** GET => user/:userId
**Get Follower/Unfollower diff:** GET => user/:userId/follower-unfollower-diff

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "spotify-followers",
   "version": "0.0.1",
   "description": "",
-  "main": "index.js",
+  "main": "server.js",
```
Comment (ddemirci) [inline server/package.json:5]: For debug

File: server/controllers/followersController.js [hunk 2]
```diff
@@ -156,4 +167,16 @@ async function updateFollowersOfUser(newFollowers, unfollowers, user) {
...
+ * Function to get followers
+ * 
+ * @returns {Promise} resolving to the followers of user.
+ * @param {String} userId
+ */
+async function getFollowers(userId){
+    const accessToken = await getAccessToken();
+    const currentFollowers = await requestFollowers(accessToken, userId);
+    return currentFollowers;
 }
```
Comment (ddemirci) [inline server/controllers/followersController.js:182]: Might be used in **getFollowersUnfollowers** controller function.

**Review by kuzdogan — APPROVED** [review]
Looks good!

File: server/controllers/followersController.js [hunk 3]
```diff
@@ -36,7 +36,18 @@ exports.getFollowersUnfollowers = function (req, res) {
 
 }
 
-
+// GET /user/:userId
```
Comment (barisucakturk) [inline server/controllers/followersController.js:39]: I think we should add '/followers' at the tail for a better understanding of what the controller does. I mean the naming seems to get only user data with userId

File: server/controllers/followersController.js (hunk 2 above)
Comment (barisucakturk) [reply server/controllers/followersController.js:182]: yepp

File: server/controllers/followersController.js (hunk 3 above)
Comment (ddemirci) [reply server/controllers/followersController.js:39]: Strongly agree ! We should convert **/user/:userId** into /user/:userId/followers

**Pull request merged**